server/sockets/list.js

```diff
@@ -0,0 +1,24 @@
+const { ItemActionTypes } = require('../common/variables');
+
+/* WS postfix stands for Web Socket, to differentiate
+ * this from controllers naming convention
+ */
+const addItemToListWS = socket => {
+  socket.on('listRoom', room => {
```

Comment:
1. Could we move joining and leaving rooms listeners to socket index, and not repeat this in every specific event (addItemToListWS)?
```
 ioInstance.on('connection', socket => {
    const {
      request: { user }
    } = socket;
    const userExist = user || false;

     if (!userExist) {
      return;
    }

socket.on('listRoom', room => {
ocket.join(room);
  });
socket.on('leavingRoom', listId => {
    socket.leave(`list-${listId}`);
  });

    addItemToListWS(socket);
  });
```
2. If we have leaveRoom event it'd be good to have joinRoom event not listRoom or joinListRoom, leaveListRoom